src/main/java/world/bentobox/bentobox/Settings.java

```diff
@@ -125,6 +125,11 @@
     @ConfigEntry(path = "panel.filler-material", since = "1.14.0")
     private Material panelFillerMaterial = Material.LIGHT_BLUE_STAINED_GLASS_PANE;
 
+    @ConfigComment("Defines how long player heads are stored into local cache before skin is required again.")
+    @ConfigComment("Value 0 will not clear cache until server restart.")
+    @ConfigEntry(path = "panel.head-cache-time", since = "1.14.1")
+    private long playerHeadCacheTime = 36000;
```

Comment:
Seconds? Minutes? Ticks? Please add that to the comment 😉

Reply:
It's milliseconds, but I recommend using minutes for cache length in this config.

Reply:
Also 36000 is only 36 seconds, so the cache time is very short. I recommend 5 or 6 minutes, which is what I think you were going for (360,000)

Reply:
Actually, the reason I made a cache was to reduce lag. It also enables refresh of skins should a player change them. With this new approach, I think something like a 1 hour or longer time would be fine.

Reply:
Value now will store minutes.
Default is set to 1h.